Notes.org

```diff
@@ -0,0 +1,51 @@
+* xio 0.13.0
+
+** open tracing
+
+*** set proper semantic tags
+
+**** https://github.com/opentracing/specification/blob/master/semantic_conventions.md
+
+** move concrete trailhead classes into http package
+
+** remove dead build tools
+
+** clean up javadoc linting errors
+
+** add http Recipes
+
+* speeding up maven invocation
+
+** drip
+
+*** https://github.com/ninjudd/drip
```

Comment:
Nailgun is old, drip is new. Honestly I'm not seeing terrible JVM startup performance on my laptop so the point is moot.